README.md

```diff
@@ -16,6 +16,7 @@ git submodule update --init --recursive
 cd scripts
 ./install_deps_ubuntu.sh
 cd ..
+pip3 install wheel setuptools
```

Comment:
Unfortunately we can't do that, because `wheel` and `setuptools` are required to successfully build `python-rocksdb`, so moving those to `requirements.txt` would break installing the deps. Your mileage may vary because of the differences in how `pip` is packaged across distros, but I've seen such behaviour on macOS and Arch-based distros

Reply:
It's possible to create another `requirements.txt` file, but I think that there is no point for that. Since `pip` analyzes transactions, it's impossible to install `wheel` and `setuptools` packages and start using them with one command, so we still must use two commands. Also having two `requirements.txt` files that depend on each other might be very confusing. I'd prefer to use multiple `requirements.txt` files only for different purposes (e.g. requirements for developers and users).